src/xmipp/applications/scripts/coordinates_consensus/coordinates_consensus.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python
```

Comment:
not python3 (multiple occurrences)?

Reply:
I guess it is equivalent now. Right? It is by default

Reply:
Right. Thx!